utils/mock/mock_executor.py

```diff
@@ -133,8 +133,9 @@ def _rollbackCommand(self, name):
     def _scrubLvmCommand(self):
         o, e, r = exxec.processToStringsWithResult(self.mainCommand() + ["--scrub", "all"])
         if r != 0:
-            raise utils.mock.mock_execution_exception.MockExecutionException("Build chroot is locked, please restart "
-                                                                             "the testsuite.")
+            failed = "Build chroot is locked, please restart the testsuite."
```

Comment:
What exactrly should change the chage from rise to log?
Afaik the raise was correctly killing the suite in case that chroot was useless. This wil lead to thausands of false failures.
But  Los Andrlos will have last word.

Reply:
The 'raise' part is still there (line 138), but the failure is logged before it kills the suite.

Reply:
I see. Yup I saw the rise later.. So the change was only for the log?
What the ```tu.passed_or_failed(self, False, failed)``` will do in this case?
Please split the refactoring and this chage.

Reply:
@judovana https://github.com/andrlos/jdk-rpms-specification-framework/issues/3 <- is the issue that is solved by this PR

Reply:
also, this change has already been thoroughly tested and works smoothly.